homeassistant/components/system_bridge/sensor.py

```diff
@@ -283,15 +308,16 @@ async def async_setup_entry(
             )
 
     displays = []
-    for display in coordinator.data.display.displays:
-        displays.append(
-            {
-                "key": display,
-                "name": getattr(coordinator.data.display, f"{display}_name").replace(
-                    "Display ", ""
-                ),
-            },
-        )
+    if coordinator.data.display.displays is not None:
+        for display in coordinator.data.display.displays:
+            displays.append(
+                {
+                    "key": display,
+                    "name": getattr(
+                        coordinator.data.display, f"{display}_name"
+                    ).replace("Display ", ""),
+                },
+            )
```

Comment:
For a more succinct patch, you could consider
```suggestion
    for display in (coordinator.data.display.displays or []):
        displays.append(
            {
                "key": display,
                "name": getattr(coordinator.data.display, f"{display}_name").replace(
                    "Display ", ""
                ),
            },
        )
```